packages/multiple-choice/configure/src/main.jsx

```diff
@@ -125,6 +125,7 @@ const Design = withStyles(styles)(props => {
             onChange={onTeacherInstructionsChanged}
             imageSupport={imageSupport}
             nonEmpty={false}
+            toolbarOpts={{ position: model.toolbarEditorPosition === 'top' ? model.toolbarEditorPosition : 'bottom' }}
```

Comment:
I would rather save this in a variable, since you're using it 4 times.
I think you can have something like:

```
const toolbarOpts = {};

switch (model.toolbarEditorPosition) {
   case 'top':
     toolbarOpts.position = 'top';
     break;
   default:
      toolbarOpts.position = 'bottom';
       break;
}
```

before the Content's declaration.